docs/build/tutorials/platform/adjusting-gas-price-during-high-network-activity.md

```diff
@@ -0,0 +1,110 @@
+| B           | 40 GWEI     | 40 GWEI     | 40 GWEI    | 0 GWEI        | 40 GWEI     |
+
+Look at transactions **A** and **B** (the bottom two transactions). In these scenarios, it looks like transaction B is paying a higher tip, however, this depends on the base fee of the block where the transactions are included. The effective tip of A is more than that of B. So, if both of these transaction competes for being included in the next block, then the validators would prioritize transaction A since it pays a higher effective tip.
+
+
+## Why my Transaction is on Hold or Failing?
+
+If your transaction is failing and giving an error - `transaction underpriced`, then the max fee cap of your transaction must be less than the minimum base price that the network supports (as of now, it's 25 nAVAX or GWEI). Although the base fee is automatically estimated in wallets like Metamask, you can try increasing the max fee cap in the wallet.
+
+During a period of heavy congestion on the network, all submitted transactions can't be included in the same block, due to the block's gas limit. So, validators choose transactions giving higher priority to transactions with the highest effective tips. Your transaction will have to wait until the effective tip is highest among the pending transactions.
+
+Another reason for pending transactions is the max fee cap being below the current base fee that the network is charging. In this case, you nat need to increase the max fee cap of the transaction.
+
+These fee adjustments can be made through wallets like MetaMask.
+
+## Adjusting Gas Fees Before Submitting the Transaction
+
+You may not need to edit the gas fees on normal days. This is only required if there is heavy congestion on the network, and the base fees are frequently fluctuating.
+
+1. Let's create a sample transaction on Avalanche Mainnet, in which we will be sending 0.1 AVAX to a receiver using the MetaMask. By clicking **Next** we can review gas fees and the amount which we want to send.
+
+![](/img/dynamic-fees-adjustment-1.png)
+
+
+2. On the review page, you can see the estimated priority and max fee for this transaction. Now click on **EDIT**, to adjust these fees according to network requirements.
+
+![](/img/dynamic-fees-adjustment-2.png)
+
+
+3. On this page, you can edit the priority fee (gas tip cap) and max fee (max fee cap) using the advanced options or you can use the MetaMask's inbuilt gas estimation. You can estimate the max fee as shown on [snowtrace](https://snowtrace.io/gastracker) which represents the average max fee over the last 3 seconds. For more detailed statistics, you can have a look [here](https://stats.avax.network/dashboard/c-chain-activity/).
+
+![](/img/dynamic-fees-adjustment-3.png)
+
+4. If the network activity is high, you have to edit the priority and max fees accordingly, as given on snowtrace. Consider the example below, where the average max fee is 30 GWEI (nAVAX).
```

Comment:
Let's get rid of this reference to snowtrace's gastracker. This is not necessary and we should recommend that users simply set the max fee cap to what they are willing to pay and set the tip cap what they are willing to pay above the base fee.

Let's recommend 100gwei and 10gwei as examples/defaults and make it clear that if their transaction is stuck, they can check snowtrace to see the base fee of the last block and increase the fees on their transaction if they are willing to pay at the higher rate.